app/grandchallenge/publications/templates/publications/publication_list.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+{% load static %}
+{% load publication_extras %}
+{% load guardian_tags %}
+{% load url %}
+{% load bleach %}
+
+
+{% block title %}Publications - {{ block.super }}{% endblock %}
+
+{% block breadcrumbs %}
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item active" aria-current="page">Publications</li>
+    </ol>
+{% endblock %}
+
+{% block content %}
+        {% include "grandchallenge/partials/filters.html" with filter=filter filters_applied=filters_applied %}
+    <div>
+        <h2>Publications</h2>
+    </div>
+    <div class="d-flex justify-content-between my-3">
+        <h6 class="ml-1 mt-2">{{ num_publications }} publications | {{ num_citations }} citations </h6>
+        <a class="btn btn-primary" href="{% url 'publications:create' %}" role="button">Add a publication</a>

```

Comment:
Only show the button if the user has the `add_publication` permission (note this is the global permission, so checked differently to the object permissions).